src/java.desktop/share/native/libawt/java2d/SurfaceData.c

```diff
@@ -80,6 +81,8 @@ Java_sun_java2d_SurfaceData_initIDs(JNIEnv *env, jclass sd)
 {
     jclass pICMClass;
 
```

Comment:
Looks like we added trace logs in common code this also i am removing under JDK-8260640